composer.json

```
@@ -68,7 +68,7 @@
 		"tomasvotruba/cognitive-complexity": "^0.2.3",
 		"tomasvotruba/type-coverage": "^0.3.1",
 		"typo3/cms-fluid-styled-content": "^11.5.4 || ^12.4.2",
-		"typo3/coding-standards": "^0.6.1",
+		"typo3/coding-standards": "^0.6 || ^0.8",
```

For the development dependencies, please let's use the exact version:

```suggestion
		"typo3/coding-standards": "^0.6.1 || ^0.8.0",
```